consensus/carnot-engine/src/lib.rs

```diff
@@ -341,16 +347,20 @@ impl<O: Overlay> Carnot<O> {
     pub fn latest_committed_blocks(&self) -> Vec<BlockId> {
         let mut res = vec![];
         let mut current = self.latest_committed_block();
-        while current != self.genesis_block() {
+
+        while res.len() < LATEST_COMMITTED_BLOCKS_LIMIT {
```

Comment:
Maybe we can just add a flag argument for the limit instead of the hardcoded constant?

Reply:
One option would be to pass limit when initializing the engine, or the function `latest_committed_blocks` could take additional param, and then http api would pass a Carnot::Info message with the limit?

Reply:
Yeah, I meant adding a parameter to the `latest_comitted_blocks` method, something like:
```rust
fn latest_comitted_blocks(&self, limit: Option<usize>) -> Vec<BlockId> {
    let limit = limit.unwrap_or_else(|| self.safe_blocks.len());
    ...
}
```
?

Reply:
Got it, will update.